docs/admin-panel/hierarchy/ERC721/features.md

```diff
@@ -0,0 +1,64 @@
+
+#### Consist of several contracts:
+- **Access Control** - provides the ability to restrict or grant permission for specific addresses to interact with a smart contract. _(see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/access-control))_
+- **Burnable** -  allows for a token to be destroyed, reducing the overall supply and potentially increasing the value of remaining tokens _(see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Burnable))_
+- **Enumerable** - allows the contract to be queried for a list of all token holders and their token balances _(see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Enumerable))_
+- **Royalty** - allows for a portion of the sale price to be paid to the original creator of the NFT each time it is resold
+- **BaseUrl** - allows the NFTs to have a URI pointing to a public location where additional information and resources can be stored
+- **MetaData** - provides tiny key/values storage for metadata associated with an NFT.
+
+Overall, the SIMPLE feature provides a comprehensive set of tools for the creation and management of NFTs, serving as the foundation for all other features offered by our platform.
+
+
+### BLACKLIST
+
+The **Blacklist** feature is a security mechanism that **restrict** certain addresses from **transfering NFT**. These addresses are typically considered to be bad actors and are unable to use the NFTs they possess if they are added to blacklist. This helps to maintain the integrity of the contract and ensure that NFTs are not being used maliciously.
+
+
+### WHITELIST
+
+The **Whitelist** feature in NFTs works as the opposite of the Blacklist. While the blacklist restrict specific addresses from transfering the NFT, the Whitelist **allows only a specific set of addresses transfer the NFT**.
+
+
+### PAUSABLE
+
+The **Pausable** feature in smart contracts allows the contract owner or authorized parties to **temporarily pause the execution of the contract**. This feature is particularly useful when the marketing team starts selling NFTs, but the programming side is not yet ready to support the transaction process. It can also be used to address unexpected bugs or errors that may arise during the lifetime of the contract, without affecting the overall integrity of the contract.
+
+One practical example of its use is in the sale of mystery boxes. If the NFTs within the mystery boxes are not yet available, the contract can be paused until they are ready. This ensures that the users cannot access the NFTs until the contract is ready, adding an extra layer of security and control to the process. The Pausable feature of ERC721 contracts allows for greater flexibility and control, making it an important component for any project utilizing NFTs.
```

Comment:
adding an extra layer of security and control to the process
->
allowing the marketing team to win time for developers or something like this